prow/cmd/deck/static/spyglass/spyglass.ts

```diff
@@ -99,6 +113,39 @@ function toggleExpansion(bodyId: string, expanderId: string): void {
   }
 }
 
+// given a string containing ansi formatting directives, return a new one
+// with designated regions of text marked with the appropriate color directives,
+// and with all unknown directives stripped
+function ansiToHTML(orig: string) {
+  // Given a cmd (like "32" or "0;97"), some enclosed body text, and the original string,
+  // either return the body wrapped in an element to achieve the desired result, or the
+  // original string if nothing works.
+  function annotate(cmd: string, body: string, orig: string) {
```

Comment:
:robot: `Avoid implicit any by always specifying return types` 

```suggestion
  function annotate(cmd: string, body: string, orig: string): string {
```